src/zkevm_specs/evm/execution/storage.py

```diff
@@ -0,0 +1,104 @@
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+from ..table import CallContextFieldTag, TxContextFieldTag
+
+COLD_SLOAD_COST = 2100
```

Comment:
move all the gas constants into a single file?

Reply:
I try moving to "src/zkevm_specs/params.py" and use 
```
from ......params import (
    COLD_SLOAD_COST,
    WARM_STORAGE_READ_COST,
    SLOAD_GAS,
    SSTORE_SET_GAS,
    SSTORE_RESET_GAS,
    SSTORE_CLEARS_SCHEDULE,
)
```
but didn't make it.

encounter an error with `ImportError: attempted relative import beyond top-level package`

so I just put it as "src/zkevm_specs/evm/execution/params.py". see 8df10c0337ba178f52af59b3752a0ba70c1c7057